app/services/db/sp_cost/add_sp_cost.rb

```diff
@@ -13,7 +13,9 @@ class SpCostTypeError < StandardError; end
         acuant_result
         lexis_nexis_resolution
         lexis_nexis_address
+        mock_resolution
```

Comment:
We were previously creating SP cost records for mock resolutions as `lexis_nexis_resolution`. This PR now starts sending `mock_resolution` in those cases.